doc/quickstart/deegree_quickstart.rst

```diff
@@ -45,46 +46,36 @@ Load a workspace
    You can see which workspace is currently active (only one can be active at a time) workspace, inactive locally available workspaces and offical workspaces which can be imported if you have a internet connection. 
-#. Press start on the *deegree-inspire-workspace*.
-
-This workspace includes an example configuration for INSPIRE View- and Download-Services.
-It consists of a transactional WFS (2.0.0 and 1.1.0) for all Annex I Data Themes
-and a WMS (1.3.0 and 1.1.1) which serves three layers of those Annex I Data 
-Themes.
-Included are some harmonized dutch administrative units, cadastral parcels and 
-addresses.
-The WFS in this example acts as an INSPIRE Download service (direct access) and 
-outputs valid, harmonized INSPIRE GML.
-Additionally it supports multiple request types and queries.
+#. Press start on the *deegree-workspace-utah-light*.
 
-.. tip::
-   This workspace loads harmonized INSPIRE GML from local files.
-   This can easily changed to PostGIS, Oracle Spatial or Microsoft SQL Server as 
-   backend.
+The demo workspace contains a web mapping setup based on data from the state of Utah.
+It contains a WMS configuration (1.3.0 and 1.1.1) consisting of vector layers and a variety of render styles. 
+Additionally, a WFS (2.0.0, 1.1.0 and 1.0.0) is configured that allows to access the raw vector data, backed by shapefiles, in GML format.
 
+.. tip::
+   For additional configuration examples including raster data, 
+   you can import the workspace *deegree-workspace-utah* through the workspace management (internet connection is required).
 
 View a map
 ==========
 
 #. Click on *'see layers'* to open a simple map-client. The service is configured to visualize the data and a basemap (rendered by `OpenStreetMap <https://www.openstreetmap.org/>`_).
 
-#. Click on the **+** on the right side to open a layer tree showing all available  layers. You can activate the INSPIRE layer served by the deegree WMS here.
+#. Click on the **+** on the right side to open a layer tree showing all available  layers. You can activate the a variety of layers served by the deegree WMS here.
 
 .. tip::
    The map client is based on `OpenLayers <https://openlayers.org/>`_. 
    You can pan through the map holding the left mouse button and drag the map.
    Zooming is possible via mouse wheel or the navigation elements on the left side.
 
-More information and step-by-step tutorials about the INSPIRE workspace can be 
-found in the deegree `documentation <https://download.deegree.org/documentation/3.4.1/html/lightly.html#example-workspace-1-inspire-network-services>`__.
-
-
 What next?
 ==========
 
 This was just a short introduction to the features of deegree 3. 
 There is more information in the demo installations and on the deegree website.
 
-  * deegree online demo - Available here: https://www.deegree.org/demo/
-
   * deegree website - Available here: https://www.deegree.org
```

Comment:
The syntax for bullets:
The bullet is at the first column of the line.
There are no blank lines in between when the bullet is at the same level.

```
* b1
* b2
* b3

   * b3.1
   * b3.2

* b4
```